### DISCUSSION TITLE
Online demos & connection to documentation

### DISCUSSION THREAD

**Discussion by emilymbender** [delph-in/docs#138] [Team Posts]
Dear all,

I was chatting with someone from ACL last week who was interested in finding the ERG demo, and struggled to do so, not least because the old demo appears to be down.

I suggest we do the following:

1) Re-direct erg.delph-in.net to something with a live demo. Perhaps https://delph-in.github.io/delphin-viz/demo/ ?  (The only drawback I see there is that the delphin-viz site isn't ERG specific.)

2) Update the version of the ERG available through that site. (The latest is 2018.)

3) Update the delphin-viz site to include pointers to:
   * https://delph-in.github.io/docs/home/Home/
   * https://delphinqa.ling.washington.edu/

Thoughts?

Emily

**Comment by oepen** [comment]
i just restarted the ERG on the UiO server and will try to keep an eye on it for at least the immediate future; please let me know in case something appears non-functional.  i still like having this version around, in part because it serves as a reference for the EDSs used in the 2019 and 2020 CoNLL shared tasks (and in part because there is a doctoral student at UiO working in this space).  but i have added a link pointing to the newer Delphin-Viz site on GitHub near the top.

**Comment by olzama** [comment]
I think it would be very good to keep the delph-in viz demo up, up to date, and linked to from other places. The main question, as usual, is who would be responsible for maintaining it. (I note here that the old demo was unfortunately down for _years_, at least I think so. Glad to hear it is back up. What was usually happening to me is, I would try to access it, remember it was down, then remember there was delphin-viz, then look for that.)

**Comment by emilymbender** [comment]
Thanks, @oepen for getting that reinstated -- and for the link.

As for delphin-viz, I understand it's hosted at UW, which is great. But we definitely need someone who can maintain it (and do things like add more recent versions, add links, etc). That person doesn't have to be a UW person -- I can definitely offer accounts on the relevant servers. Is anyone interested?

**Comment by olzama** [comment]
I think actual maintenance (ensuring the page is up and all functionality
is working) is possible only at the place where it’s hosted?

On Thu, Jul 20, 2023 at 3:33 PM emilymbender ***@***.***>
wrote:

> Thanks, @oepen <https://github.com/oepen> for getting that reinstated --
> and for the link.
>
> As for delphin-viz, I understand it's hosted at UW, which is great. But we
> definitely need someone who can maintain it (and do things like add more
> recent versions, add links, etc). That person doesn't have to be a UW
> person -- I can definitely offer accounts on the relevant servers. Is
> anyone interested?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/discussions/138#discussioncomment-6499656>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACTURKRXVTSER6BMAEJKJWLXREXRHANCNFSM6AAAAAA2QVWMAI>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
-- 
Olga

**Comment by emilymbender** [comment]
Well, anyone can be "at" UW with an account on the relevant machine. (And if the machines are down, that's a more general problem that we'd be on top of anyway.)

**Comment by olzama** [comment]
It's not always as general as the machines being entirely down; certificates expire, updates don't get installed... email doesn't get configured correctly :). I don't know which of these things are relevant to delphin-viz though. I suppose the pesky email issue doesn't apply, which is already good :). 

But in general it seems to me like it's not easy to maintain a web service where you are not an actual server admin...

**Comment by emilymbender** [comment]
Brandon (our sysadmin) and I can help with those kids of things. Certificates are definitely already on our end, etc. What is needed is someone who can go in and do things like add an up to date version of the ERG. I'd be happy to work with that person, but can't volunteer for that job.
